src/main/java/com/google/sps/tool/ArrayUtil.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.tool;
+
+import java.util.Comparator;
+import java.util.Arrays;
+
+public class ArrayUtil {
+
+  public static int[] argsort(float[] a) {
+    return argsort(a, true);
+  }
+
+  public static int[] argsort(final float[] a, final boolean ascending) {
```

Comment:
Add a comment referencing Numpy's argsort. Otherwise this function's name is unclear.

Reply:
done

Reply:
This is [from](https://github.com/deeplearning4j/nd4j/blob/8f005bcecb240d1fbb83b9d390ad801d1d3b6933/nd4j-common/src/main/java/org/nd4j/linalg/util/ArrayUtil.java#L1230) nd4j, right? I assume it is copied here to avoid having to import all of ArrayUtils from that library. No?

Reply:
Very important that you give attribution to code you didn't write, including the correct license (in this case, New BSD).

https://code.google.com/archive/p/array4j/

Reply:
The license should be copied into a header comment.